var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -50,11 +50,12 @@ class Adios(AutotoolsPackage):
 
     variant('mpi', default=True, description='Enable MPI support')
     variant('infiniband', default=False, description='Enable infiniband support')
+    variant('mxml', default=False, description='Use external MXML library')
```

Comment:
> It is extremely tiresome to fix N different dependent packages just to get the 1 package actually built!

i know the pain, and I came across similar issues while building `dealii` which eventually depends on another 20+ packages. My take on this problem is that this is the price to pay when working with Spack. The Spack community do its best to maintain a working collection of packages across various platforms/compilers/variants/etc. And if something is broken for specific case, then it's ought to be fixed anyhow.